### PR TITLE
Calibration DB Integration

### PR DESCRIPTION
**Pull request by rkr35**
These changes allow an athlete to save their calibrated BLE tags to a device-local database. 

These changes also allow an athlete to select a preferred, default tag for workouts. 

Consequently, the Current Workout screen connects to the default tag and uses its calibrated threshold for lap counting.

### CODE REVIEW

File: app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/lapcounter/LapCounterService.java [hunk 1]
```diff
@@ -101,4 +106,10 @@ public void reset() {
         initCallbacks();
         mReceiver.attach(this);
     }
+
+    public void updateThreshold(String mac) {
+        DeviceRepository repo = new DeviceRepository(getApplication());
```
Comment (jdy27) [inline app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/lapcounter/LapCounterService.java]: Is there a reason why you use the DeviceRepository as opposed to the DeviceViewModel?

Comment (rkr35) [reply app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/lapcounter/LapCounterService.java]: The LapCounterService is a backend module with no UI component, so there is no UI data for the DeviceViewModel to hold.

File: app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/CurrentWorkoutActivity.java [hunk 2]
```diff
@@ -33,6 +33,8 @@
 import static edu.drexel.lapcounter.lapcounter.backend.ble.RSSIManager.DEFAULT_DELTAS_WINDOW_SIZE;
 import static edu.drexel.lapcounter.lapcounter.backend.ble.RSSIManager.DEFAULT_MOVING_AVERAGE_SIZE;
 
+//import android.support.v4.content.LocalBroadcastManager;
```
Comment (pfh28) [inline app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/CurrentWorkoutActivity.java]: we should probably remove this at some point

**Comment by pfh28** [comment]
Calibration feels much more meaningful with this indication that it's maintained, and counting is working to a reasonable degree on the current workout screen. Looks good!

**Pull request merged**